lib/griddler/email.rb

```diff
@@ -37,11 +37,12 @@ def config
     end
 
     def recipients(type)
-      params[type].to_a.map { |recipient| extract_address(recipient) }
+      params[type].to_a.map { |recipient| extract_address(recipient) }.compact
```

Comment:
So I pushed a commit that added that in but realized that the test I wrote didn't actually break... After digging in a bit, I think I misunderstood how compact works:

```
irb(main):001:0> [nil].compact
=> []
irb(main):002:0> [nil,nil].compact
=> []
irb(main):003:0> [nil, 1,2,3].compact
=> [1, 2, 3]
```

So I think we're good w/o the `|| []` :sweat_smile: :ok_hand: